sensors/nxt_i2c_sensor_defs.c

```diff
@@ -25,6 +25,36 @@
 #include "ms_ev3_smux.h"
 #include "ms_nxtmmx.h"
 
+/*
+ * convert raw data of LEGO NXT Temperature Sensor to Celsius
+ */
+static int lego_nxt_temp_celsius_scale(void *context,
```

Comment:
I've thought about this a bit more. I would like to change `raw_max` and `raw_min` to s32. There are only a couple sensors that actually use 32-bit raw data and they are signed. All unsigned cases are 16-bit or less, so will fit in s32.